base-hack/include/common_structs.h

```diff
@@ -228,7 +228,9 @@ typedef struct playerData {
 	/* 0x155 */ char control_state_progress;
 	/* 0x156 */ char unk_156[0x16A-0x156];
 	/* 0x16A */ unsigned char rgb_components[3];
-	/* 0x16D */ char unk_16D[0x18A-0x16D];
+	/* 0x16D */ char unk_16D;
+	/* 0x16D */ char unk_16E; // shadow width or something
```

Comment:
Can you change this to 0x16E and 0x16F in the comments

Reply:
Absolutely! Good catch :D